lib/matplotlib/axes/_base.py

```diff
@@ -2609,6 +2609,26 @@ def use_sticky_edges(self, b):
         self._use_sticky_edges = bool(b)
         # No effect until next autoscaling, which will mark the Axes as stale.
 
+    def get_xmargin(self):
+        """
+        Retrieve autoscaling margin of the x-axis.
+
+        Returns
+        -------
+        xmargin : float
```

Comment:
Since the docstring for the `set_xmargin` method contains quite a lot of explanation, I think it might be good to add a "see also" referencing that, similar to what is done in `get_anchor`.

Similarly for the other two.